mikrotik/internal/test_helpers.go

```diff
@@ -0,0 +1,40 @@
+package internal
+
+import (
+	"fmt"
+)
+
+var ipCounter uint = 0xC0A80001      // 192 = C0, 168 = A8, 0 = 00, 1 = 01
+var ipRangeCounter uint = 0xAC100001 // 172 = AC, 16 = 10, 0 = 00, 1 = 01
+var macCounter = 0
+var dnsCounter = 0
+
+func GetNewIpAddr() string {
```

Comment:
I'd prefer to not use these new functions until an ephemeral test environment is in place. I know you mentioned in discord that your other branch depends on these changes, but I'm hesitant to test any branch against my router that doesn't use very specific IPs, mac addresses, etc.

Could we merge the change to add a Github action with the acceptance tests first?